fdbclient/NativeAPI.actor.cpp

```diff
@@ -255,16 +255,27 @@ void DatabaseContext::getLatestCommitVersions(const Reference<LocationInfo>& loc
 
 	std::map<Version, std::set<Tag>> versionMap; // order the versions to be returned
 	for (int i = 0; i < locationInfo->locations()->size(); i++) {
+		int updatedVersionMap = 0;
+		Version commitVersion = invalidVersion;
+		Tag tag = invalidTag;
 		UID uid = locationInfo->locations()->getId(i);
 		if (ssidTagMapping.find(uid) != ssidTagMapping.end()) {
-			Tag tag = ssidTagMapping[uid];
+			tag = ssidTagMapping[uid];
```

Comment:
This is doing a second lookup. You can avoid it by saving the return value of `ssidTagMapping.find(uid)`.